frontend/Dockerfile.playwright

```diff
@@ -9,6 +9,9 @@ WORKDIR /frontend
 
 COPY package.json .
 
+# Copy the pnpm-lock.yaml from the root of the monorepo
+COPY ../pnpm-lock.yaml ./pnpm-lock.yaml
+
```

Comment:
Why does this copy `pnpm-lock.yaml` file from the parent directory when the `ci_cd.yml` file has copied it into `frontend/`? Also how is this Dockerfile able to access files from one level up? As per my understanding, it would not be part of the Docker context.

Reply:
Updated to copy from frontend directory to docker.